Volume/Views/MainView/HomeList.swift

```diff
@@ -75,66 +100,95 @@ struct HomeList: View {
+                                    .resizable()
+                                    .aspectRatio(contentMode: .fill)
+                                HStack(alignment: .top) {
+                                    Text("Your\nWeekly\nDebrief")
+                                        .font(.begumRegular(size: 18))
+                                        .foregroundColor(.volume.orange)
+                                        .padding([.leading])
+                                    Spacer()
+                                    Image("right-arrow")
+                                        .padding([.trailing])
+                                }
+                            }
+                        }
                     }
-                case .reloading(let results), .results(let results):
-                    ForEach(results.followedArticles) { article in
-                        ArticleRow(article: article, navigationSource: .followingArticles)
-                            .padding([.leading, .trailing])
+                }
+                .frame(height: 92)
+                .padding([.horizontal])
+                .shadow(color: .volume.shadowBlack, radius: 5, x: 0, y: 0)
+                Group { // Articles from Followed Publications
+                    Header("Following")
+                        .padding([.leading, .trailing])
+                    switch state {
+                    case .loading:
+                        ForEach(0..<5) { _ in
+                            ArticleRow.Skeleton()
+                                .padding([.leading, .trailing])
+                        }
+                    case .reloading(let results), .results(let results):
+                        ForEach(results.followedArticles) { article in
+                            ArticleRow(article: article, navigationSource: .followingArticles)
+                                .padding([.leading, .trailing])
+                        }
                     }
+                    Spacer()
+                    VolumeMessage(message: .upToDate)
+                        .padding(.top, 25)
+                        .padding(.bottom, -5)
                 }
-
-                Spacer()
-
-                VolumeMessage(message: .upToDate)
-                    .padding(.top, 25)
-                    .padding(.bottom, -5)
-
                 Spacer()
-
-                Header("Other Articles").padding()
-                switch state {
-                case .loading:
-                    // will be off the page, so pointless to show anything
-                    Spacer().frame(height: 0)
-                case .reloading(let results), .results(let results):
-                    ForEach(results.otherArticles) { article in
-                        ArticleRow(article: article, navigationSource: .otherArticles)
-                            .padding([.bottom, .leading, .trailing])
+                Group { // Articles from Non-followed Publications
```

Comment:
This doesn't need to be in a group. I think I know what you're trying to do here (organization), but you can achieve the same thing (except better) by dividing this into multiple private class variables like I suggested above.

Reply:
Unfortunately the SwiftUI compiler doesn't allow returning more than one View in a private class variable so they have to be wrapped in a group, especially if one of the items is wrapped in a `switch` statement